### PR TITLE
[Outreachy] Registry: Add Tags to Items #5336

### PR DESCRIPTION
**Pull request by olamideTiana**
#5336 - Add Tags to Items

Added Tags to the existing data entries

### CODE REVIEW

**Comment by olamideTiana** [comment]
@svrnm Please Kindly review sir and I'll appreciate your feedback on it

**Review by olamideTiana — COMMENTED** [review]
Add Tags to Items

File: data/registry/application-integration-collector-cloud foundry.yml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 title: Cloud Foundry
 registryType: application integration
 language: collector
-tags:
+tags: Metrics
   - collector
   - cloud foundry
```
Comment (svrnm) [inline data/registry/application-integration-collector-cloud foundry.yml:6]: "collector" and "cloud foundry" are the tags already, you don't need to add anything to those files.

**Comment by svrnm** [comment]
@olamideTiana a few hints for you what needs to be done:

In the file https://github.com/open-telemetry/opentelemetry.io/blob/main/layouts/partials/ecosystem/registry/entry.html you will need to find a place where to put the tags. To begin with there is no need to read the data from the entry yet, just use some sample values to setup the user experience for it. You can take inspiration from other registries how they display tags, e.g. https://www.npmjs.com/search?q=opentelemetry

**Comment by olamideTiana** [comment]
Ohh, sorry for the inconvenience @svrnm Thanks for shedding more light to the issue, I'll fix it now. By the way, can I close this PR

**Comment by olamideTiana** [comment]
@svrnm Okay, thank you

**Pull request closed without merging**